pkg/tfvars/internal/cache/cache.go

```diff
@@ -107,6 +110,29 @@ func cacheFile(reader io.Reader, filePath string, sha256Checksum string) (err er
 		}
 	}()
 
+	// Detect whether we know how to decompress the file
+	// See http://golang.org/pkg/net/http/#DetectContentType for why we use 512
+	buf := make([]byte, 512)
+	_, err = reader.Read(buf)
+	if err != nil {
+		return err
+	}
+
+	reader = io.MultiReader(bytes.NewReader(buf), reader)
+	fileType := http.DetectContentType(buf)
+	logrus.Debugf("content type of %s is %s", filePath, fileType)
```

Comment:
https://prow.svc.ci.openshift.org/view/gcs/origin-ci-test/pr-logs/pull/openshift_installer/2657/pull-ci-openshift-installer-master-e2e-openstack/2837#1:build-log.txt%3A1233

the output looks like is printing the entire os.File object.

maybe `Detected content-type as <type>` ?? because we are no longer using the file to detect content.

Reply:
filePath is the filename.  I think it's useful this way, so if something goes wrong somebody can look in the cache to see which file was misdetected or whatnot.

Reply:
(that failure was old, when I had file rather than filePath in there)